src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-# pylint: disable=line-too-long, too-many-lines
+# pylint: disable=line-too-long, too-many-lines, option_length_too_long
```

Comment:
Please configure it in [linter_exclusions.yml](https://github.com/Azure/azure-cli/blob/dev/linter_exclusions.yml) to exclude it